io.quarkus:quarkus-maven-plugin error on ppcle and s390x linux

Hi there,
I am working on running quarkus-openshift test on x84, ppc64le, and s390x linux machines.
The x84 works fine to pass most tests, but for ppc64le and s390x, all tests (e.g. HttpOpenShiftIT) with ```io.quarkus:quarkus-maven-plugin``` failed with error message ```Unknown maven version output format. Expected at least one line!```.
Some combinations I tried with running results are: 
- host machine x86, Openshift Platform 4.5 on Intel/ppcle/s390x, Passed
- host machine ppcle, Openshift Platform 4.3/4.5 on ppcle/intel, failed with ```io.quarkus:quarkus-maven-plugin```
- host machine s390x, Openshift Platform 4.5 on s390x/intel, failed with ```io.quarkus:quarkus-maven-plugin```

More detailed error message is:
```
16:23:23  [ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:1.6.0.Final:build (default) on project security-https-1way: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
16:23:23  [ERROR] 	[error]: Build step io.quarkus.kubernetes.deployment.KubernetesProcessor#build threw an exception: java.lang.IllegalStateException: Unknown maven version output format. Expected at least one line!
16:23:23  [ERROR] 	at io.dekorate.utils.Maven.getVersionFromOutput(Maven.java:72)
16:23:23  [ERROR] 	at io.dekorate.utils.Maven.getVersion(Maven.java:58)
16:23:23  [ERROR] 	at io.dekorate.project.MavenInfoReader.getInfo(MavenInfoReader.java:73)
16:23:23  [ERROR] 	at io.dekorate.project.FileProjectFactory.lambda$getProjectInfo$2(FileProjectFactory.java:78)
16:23:23  [ERROR] 	at io.dekorate.project.FileProjectFactory$$Lambda$972/0000000000000000.apply(Unknown Source)
16:23:23  [ERROR] 	at java.base/java.util.Optional.map(Optional.java:265)
16:23:23  [ERROR] 	at io.dekorate.project.FileProjectFactory.getProjectInfo(FileProjectFactory.java:78)
16:23:23  [ERROR] 	at io.dekorate.project.FileProjectFactory.createInternal(FileProjectFactory.java:52)
16:23:23  [ERROR] 	at io.dekorate.project.FileProjectFactory.create(FileProjectFactory.java:41)
16:23:23  [ERROR] 	at io.quarkus.kubernetes.deployment.KubernetesProcessor.createProject(KubernetesProcessor.java:738)
16:23:23  [ERROR] 	at io.quarkus.kubernetes.deployment.KubernetesProcessor.build(KubernetesProcessor.java:252)
16:23:23  [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
16:23:23  [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
16:23:23  [ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
16:23:23  [ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
16:23:23  [ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:932)
16:23:23  [ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
16:23:23  [ERROR] 	at io.quarkus.builder.BuildContext$$Lambda$453/0000000000000000.run(Unknown Source)
16:23:23  [ERROR] 	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
16:23:23  [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2046)
16:23:23  [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1578)
16:23:23  [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
16:23:23  [ERROR] 	at java.base/java.lang.Thread.run(Thread.java:836)
16:23:23  [ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:479)
```
and when I use ```./mvnw -v``` to check the most machine mvn verion, I got:
```18:14:35  [1mApache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T15:00:29-04:00)[m
18:14:35  Maven home: /home/jenkins/.m2/wrapper/dists/apache-maven-3.6.1-bin/38pn40mp89t5c94bjdbeod370m/apache-maven-3.6.1
18:14:35  Java version: 11.0.8, vendor: AdoptOpenJDK, runtime: /home/jenkins/workspace/Grinder/openjdkbinary/j2sdk-image
18:14:35  Default locale: en_US, platform encoding: UTF-8
18:14:35  OS name: "linux", version: "3.10.0-693.17.1.el7.ppc64le", arch: "ppc64le", family: "unix"
```
and
```10:15:58  Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T12:00:29-07:00)
10:15:58  Maven home: /home/jenkins/.m2/wrapper/dists/apache-maven-3.6.1-bin/38pn40mp89t5c94bjdbeod370m/apache-maven-3.6.1
10:15:58  Java version: 11.0.8, vendor: AdoptOpenJDK, runtime: /home/jenkins/workspace/Grinder/openjdkbinary/j2sdk-image
10:15:58  Default locale: en_US, platform encoding: ANSI_X3.4-1968
10:15:58  OS name: "linux", version: "4.4.0-179-generic", arch: "s390x", family: "unix"
```

Really appreciate for suggestions to solve this problem. Thanks!

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -148,7 +148,7 @@
         <aws-alexa-sdk.version>2.32.0</aws-alexa-sdk.version>
         <azure-functions-java-library.version>1.3.0</azure-functions-java-library.version>
         <kotlin.version>1.3.72</kotlin.version>
-        <dekorate.version>0.12.7</dekorate.version>
+        <dekorate.version>0.12.8</dekorate.version>
         <maven-artifact-transfer.version>0.10.0</maven-artifact-transfer.version>
         <jline.version>2.14.6</jline.version>
         <maven-invoker.version>3.0.1</maven-invoker.version>

